parser/prism/Translator.cc

```diff
@@ -1639,7 +1641,7 @@ unique_ptr<parser::Node> Translator::translateStatements(pm_statements_node *stm
 // Usually returns the `SorbetLHSNode`, but for constant writes and targets,
 // it can can return an `LVarLhs` as a workaround in the case of a dynamic constant assignment.
 template <typename PrismLhsNode, typename SorbetLHSNode>
-unique_ptr<parser::Node> Translator::translateConst(PrismLhsNode *node) {
+unique_ptr<parser::Node> Translator::translateConst(PrismLhsNode *node, bool skipDynamicConstantWorkaround) {
```

Comment:
Could you add a blurb to the function docs that explains what this is (and why constant path op assignments need it)?

Reply:
@amomchilov Let me know what you think of the explanation above!

Reply:
Looks good!